backend/eks/kubeconfig.sh

```diff
@@ -4,12 +4,43 @@
 . ../../include/common.sh
 . .envrc
 
+
+# KUBECFG can be a kubeconfig file or a zip archive containing a tf directory for local deployments
```

Comment:
You don't need any logic in kubeconfig.sh. All that will still work the same. All you need is to handle tf.zip in clean.sh.

Reply:
Interesting. I will look at @viccuad 's PR #192 as that is apparently the larger framework.